.github/workflows/ci.yml

```diff
@@ -934,6 +998,7 @@ jobs:
       - detect-changes
       - docker-checks
       - elasticsearch-integration-tests
+      - rdbms-h2-integration-tests
```

Comment:
Please insert job names in alphabetical order here for easier conflict resolution etc.